build/common.go

```diff
@@ -60,7 +60,7 @@ func getExecutableName(os string, arch string, pluginJSONPath string) (string, e
 		exname = exename
 	}
 
-	return asExecutableName(os, arch, pluginJSONPath), nil
+	return asExecutableName(os, arch, exname), nil
```

Comment:
Does this part need to change? This will change behaviour for nested datasource executables

Reply:
Ahhh I see now - I didn't realized I completely fudged it 🤦